### PR TITLE
add case insensitive support for docs

### LINKED ISSUE #280: Not seeing guides
### Description

https://github.com/fastify/website/pull/269

I am not seeing the result of that work on https://www.fastify.io/docs/master/ . Should I be?

@lmammino

### PR DESCRIPTION
**Pull request by luisorbaiceta**
fix #280 

As discussed with @jsumners , here is the case insensitive version for the nested docs.

#### Checklist
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: src/scripts/processReleases.js [hunk 1]
```diff
@@ -83,6 +83,14 @@ async function extractTOCFromReleaseStructure (root, release) {
       docsPath: release.docsPath,
       label: release.label
     }
+
+    Object.keys(toc).forEach(key => {
+      if (key !== 'sourceFile') {
+        toc[key] = toc[key].replace('Index', 'index')
```
Comment (climba03003) [inline src/scripts/processReleases.js]: Can we use `.toLowerCase` in here? It seems like it is just a hotfix for `Index`, but not a case-insensitive fix.

**Review by climba03003 — CHANGES_REQUESTED** [review]

File: src/scripts/processReleases.js (hunk 1 above)
Comment (luisorbaiceta) [reply src/scripts/processReleases.js]: done. I tweaked the layouts as well to have a nice capitalized index in sidebar, table of contents and the breadcrumb

**Review by climba03003 — APPROVED** [review]
LGTM.

**Comment by climba03003** [comment]
It seems like `.toLowerCase` is actually breaking the site. Sorry for that advice.
@luisorbaiceta Do you have time for a quick look on it?

The breaking I mention is about the link inside the page, not the sidebar.
For example: https://www.fastify.io/docs/latest/reply/#introduction, clicking `Request's context` will redirect to wrong replace.

**Comment by luisorbaiceta** [comment]
> The breaking I mention is about the link inside the page, not the sidebar. For example: https://www.fastify.io/docs/latest/reply/#introduction, clicking `Request's context` will redirect to wrong replace.

I was afraid something like this might happen. Current implementation for cases is not very consistent. I will rollback to only tweaking the `Index` as a quick fix and see if I can find some time to rethink the logic.

**Pull request merged**